Reward - managing kudos - 1 screenshot missing (#142)
exoplatform/exo-documentation: docs/Administration.rst

.. note:: No action except "Refresh" is allowed for "admin" wallet.

.. _ManagingKudos:

===============
Managing Kudos
===============

You can manage the use of kudos in the platform from "Kudos settings" page. 

> To access to this interface, click on "Administration" menu from the top navigation, then "Reward" > "Kudos" from the drop-down menu.

Reviewer: ```suggestion
To access this interface, click on "Administration" menu from the top navigation, then "Reward" > "Kudos" from the drop-down menu.
```

> To access to this interface, you have to click on "Administration" menu from the top navigation, then "Reward" - "Kudos" from the drop-down menu.

Reviewer: ```suggestion
To access to this interface, click on "Administration" menu from the top navigation, then "Reward" > "Kudos" from the drop-down menu.
```

|image333|

> You can set the number of the allocated kudos for users to use per period of time (week, month, quarter or year). 

Reviewer: ```suggestion
You can define how many kudos users can send  per period of time (week, month, quarter or year). 
```

> You can also restrict the use of Kudos to the members of a single space. Here space membership is simply used to indicate a group permission. Authorized users will be able to send and receive kudos from any other space.

Reviewer: ```suggestion
You can also restrict the use of kudos to the members of a single space. Here, the space membership is simply used as a group permission. Authorized users will be able to send and receive kudos from any other space.
```

> You can also restrict the use of Kudos to the members of a single space. They will be allowed to send and recieve Kudos even from other space's activities. In this case, both sender and reciever must be in the specified space.

Reviewer: ```suggestion
You can also restrict the use of Kudos to the members of a single space. Here space membership is simply used to indicate a group permission. Authorized users will be able to send and receive kudos from any other space.
```

|image334|

> You can access to the list of the sent kudos from the tab "Kudos List" of the same interface. The information of the sender and reciever are available.

Reviewer: ```suggestion
You can review the list of kudos received and sent from the tab "Kudos List" of the same interface. The information of the sender and receiver are available.
```
